packages/cozy-doctypes/src/contacts/Contact.spec.js

```diff
@@ -102,4 +102,42 @@ describe('Contact model', () => {
       expect(result).toEqual('https://arya.mycozy.cloud')
     })
   })
+
+  describe('getDisplayName function', () => {
+    it("should return the contact's fullname if any", () => {
+      const contact = {
+        fullname: 'Doran Martell',
+        name: {
+          givenName: 'Doran',
+          familyName: 'Martell'
+        },
+        email: [
+          {
+            address: 'doran.martell@dorne.westeros',
+            primary: true
+          }
+        ]
+      }
+      const result = Contact.getDisplayName(contact)
+      expect(result).toEqual('Doran Martell')
+    })
+
+    it("should return the contact's primary email if no fullname", () => {
```

Comment:
Are we sure of that? 

I would use fullname > givenName + familyName > email

Reply:
This is the current specified behavior. We asked this morning to have a written specification about that. We'll update this function later if needed ;)

Reply:
I think we always have `fullname` if `givenName` and `familyName` are present but it costs almost nothing and it's obviously better. 
(We asked Thomas to clarify the order for the display name this morning)